src/cockpit/389-console/src/lib/database/suffix.jsx

```diff
@@ -773,6 +775,10 @@ export class Suffix extends React.Component {
             cmd.push("--dncache-memsize=" + this.state.dncachememsize);
             requireRestart = true;
         }
+        if (this.state._dbstate != this.state.dbstate) {
```

Comment:
Currently, if the user changes 'Backend State' to 'Refferal' but no referrals exist, the user gets 'Unwilling to perform' message in the notification.

It is a bit cryptic.
Only in the Console Log we can see:

```
cockpit.js:1 {
    "msgtype": 103,
    "msgid": 9,
    "result": 53,
    "desc": "Server is unwilling to perform",
    "ctrls": [],
    "info": "need to set nsslapd-referral before moving to referral state\n"
}
```

I wonder if it's worth adding additional validation here (or print the meaningful message in the notification).

Reply:
We can add a check for "info" in the message and print it..  I've done this in other parts of the code, but it was too big of change to do it across the board with my previous work.  Maybe we should open a new ticket for adding the "info" check when doing error notifications

Reply:
Ok, I fixed it, and I will open a ticket to make this approach consistent across the UI